content/en/users/tutorials/jupyter-datahub-virtual-machine/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Create a Virtual Machine with Jupyter and DataHub"
+type: docs
+weight: 100
+description: >
+  Step by step guide to get a Virtual Machine for Jupyter and DataHub in your
+  cloud provider
+---
+
+## Overview
+
+This tutorial describes how to start a Virtual Machine in the EGI Federation
+that runs a browser accessible jupyter server with DataHub spaces mounted. This
+setup can be used in the EGI Federation or in any other provider which
+synchronise images from AppDB but is not part of the federation.
+
+## Step 0: Requirements
```

Comment:
I would keep the title as just "Requirements"

Reply:
Fixed